installation_and_upgrade/instrument_install_latest_build_only.bat

```diff
@@ -22,9 +23,13 @@ if "%1" == "INCR" (
 )
 
 call "%LATEST_PYTHON%" "%~dp0IBEX_upgrade.py" --kits_icp_dir "%KITS_ICP_PATH%"  %SERVER_BUILD_PREFIX% --quiet %INSTALL_TYPE%
-IF ERRORLEVEL 1 GOTO :ERROR
-
+set errcode=%ERRORLEVEL%
 popd
+IF %errcode% neq 0 GOTO :ERROR
+
+call "%~dp0install_latest_isisicp.bat" 2
```

Comment:
All of the install tasks should be in the same place (in the python code), so that:
- the code is easier to understand in future
- we can decide to do/not do this step as usual
- we get the standard skip/retry/abort options if it fails
- we facilitate adding this to the normal instrument deploy script if we choose to do that in future.